settings.gradle.kts

```diff
@@ -0,0 +1,9 @@
+pluginManagement {
+    repositories {
+        jcenter()
+        maven("https://repo.spongepowered.org/maven")
+        gradlePluginPortal()
```

Comment:
`gradlePluginPortal()` includes `jcenter()`, so

```kts
repositories {
    gradlePluginPortal()
    maven("https://repo.spongepowered.org/maven")
}
```
should be equivalent. However, I believe we have most (all?) of our Gradle plugins on the Gradle plugin portal anyway.

That reminds me, I did message them to transfer all plugins to the SpongePowered account, but never got any reply...